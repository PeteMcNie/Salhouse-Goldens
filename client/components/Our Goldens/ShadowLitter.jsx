import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'

function ShadowLitter () {
  return (
    <>
      <div className='mainBody'>
        <Container>
          <Row className='row'>
            <Col>
              <h2 className='litterTitle'>Shadow&apos;s litter 2010</h2>
            </Col>
          </Row>
          <Row>
            <Col md={6} className='leftCol'>
              <Image src="images/sl1.jpg" fluid className='litterMainImage'/>
              <p className='litterMainText'>Shadow</p>
            </Col>
            <Col md={6}>
              <Image src="images/sl2.jpg" fluid className='litterMainImage'/>
              <p className='litterMainText'>Gambler<a href='https://www.mistymoorgoldens.com/'> (NZ CH Santamaria The Patriot)</a></p>
            </Col>
          </Row>
          <Row>
            <Col>
              <p>
                My thanks go to Gaye who kindly allowed me to use the lovely Gambler as sire for this litter
                and also to Sandy for her invaluable help with the whelping of the litter.
              </p>
              <p>
                On 6 and 7 April 2010 Shadow had a litter of 7 beautiful puppies, 4 males and 3 females. She
                also had another female but unfortunately that was stillborn.
              </p>
            </Col>
          </Row>

          <hr className='sectionSeparator' />

          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl3.jpg" fluid className='litterImage'/>
              <p className='litterText'></p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl4.jpg" fluid className='litterImage'/>
              <p className='litterText'></p>
            </Col>
          </Row>
          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl5.jpg" fluid className='litterImage'/>
              <p className='litterText'></p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl6.jpg" fluid className='litterImage'/>
              <p className='litterText'></p>
            </Col>
          </Row>
          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl7.jpg" fluid className='litterImage'/>
              <p className='litterText'>1 Male - tip of tail - temporary name, Xerxes; final name Lachie</p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl8.jpg" fluid className='litterImage'/>
              <p className='litterText'>2 Male - base of tail - temporary name, Spartacus; final name, Murphy</p>
            </Col>
          </Row>
          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl9.jpg" fluid className='litterImage'/>
              <p className='litterText'>3 Female - neck - temporary name, Trinity; final name, Kylo</p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl10.jpg" fluid className='litterImage'/>
              <p className='litterText'>4 Male - middle of back - temporary name, Thumper; final name, Marco</p>
            </Col>
          </Row>
          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl11.jpg" fluid className='litterImage'/>
              <p className='litterText'>5 Female - base of tail - temporary name, Flotsam; final name, Sherry</p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl12.jpg" fluid className='litterImage'/>
              <p className='litterText'>6 Male - neck - temporary name, Maximus; final name, Max</p>
            </Col>
          </Row>
          <Row>
            <Col lg={6} className='leftCol'>
              <Image src="images/sl13.jpg" fluid className='litterImage'/>
              <p className='litterText'>7 Female - tip of tail - temporary name, Tiny; final name, Amber</p>
            </Col>
            <Col lg={6}>
              <Image src="images/sl14.jpg" fluid className='litterImage'/>
              <p className='litterText'>Sleeping puppies nearly 3 weeks old.</p>
            </Col>
          </Row>
        </Container>
      </div>
    </>
  )
}

export default ShadowLitter
