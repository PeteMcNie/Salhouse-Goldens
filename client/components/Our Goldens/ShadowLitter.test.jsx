import React from 'react'
import { render } from '@testing-library/react'

import ShadowLitter from './ShadowLitter'

test('Shadown component matches snapshot', () => {
  const { container } = render(<ShadowLitter />)
  expect(container).toMatchSnapshot()
})
