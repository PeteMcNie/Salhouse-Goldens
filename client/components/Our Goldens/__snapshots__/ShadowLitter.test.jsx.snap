// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shadown component matches snapshot 1`] = `
<div>
  <div
    class="mainBody"
  >
    <div
      class="container"
    >
      <div
        class="row row"
      >
        <div
          class="col"
        >
          <h2
            class="litterTitle"
          >
            Shadow's litter 2010
          </h2>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-md-6"
        >
          <img
            class="litterMainImage img-fluid"
            src="images/sl1.jpg"
          />
          <p
            class="litterMainText"
          >
            Shadow
          </p>
        </div>
        <div
          class="col-md-6"
        >
          <img
            class="litterMainImage img-fluid"
            src="images/sl2.jpg"
          />
          <p
            class="litterMainText"
          >
            Gambler
            <a
              href="https://www.mistymoorgoldens.com/"
            >
               (NZ CH Santamaria The Patriot)
            </a>
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col"
        >
          <p>
            My thanks go to Gaye who kindly allowed me to use the lovely Gambler as sire for this litter and also to Sandy for her invaluable help with the whelping of the litter.
          </p>
          <p>
            On 6 and 7 April 2010 Shadow had a litter of 7 beautiful puppies, 4 males and 3 females. She also had another female but unfortunately that was stillborn.
          </p>
        </div>
      </div>
      <hr
        class="sectionSeparator"
      />
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl3.jpg"
          />
          <p
            class="litterText"
          />
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl4.jpg"
          />
          <p
            class="litterText"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl5.jpg"
          />
          <p
            class="litterText"
          />
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl6.jpg"
          />
          <p
            class="litterText"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl7.jpg"
          />
          <p
            class="litterText"
          >
            1 Male - tip of tail - temporary name, Xerxes; final name Lachie
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl8.jpg"
          />
          <p
            class="litterText"
          >
            2 Male - base of tail - temporary name, Spartacus; final name, Murphy
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl9.jpg"
          />
          <p
            class="litterText"
          >
            3 Female - neck - temporary name, Trinity; final name, Kylo
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl10.jpg"
          />
          <p
            class="litterText"
          >
            4 Male - middle of back - temporary name, Thumper; final name, Marco
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl11.jpg"
          />
          <p
            class="litterText"
          >
            5 Female - base of tail - temporary name, Flotsam; final name, Sherry
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl12.jpg"
          />
          <p
            class="litterText"
          >
            6 Male - neck - temporary name, Maximus; final name, Max
          </p>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="leftCol col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl13.jpg"
          />
          <p
            class="litterText"
          >
            7 Female - tip of tail - temporary name, Tiny; final name, Amber
          </p>
        </div>
        <div
          class="col-lg-6"
        >
          <img
            class="litterImage img-fluid"
            src="images/sl14.jpg"
          />
          <p
            class="litterText"
          >
            Sleeping puppies nearly 3 weeks old.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
